### PR TITLE
Pete stack part two the typo fixes

### PR DESCRIPTION
**Pull request by nelsonic**
@jruts, thanks for noting all the typos and grammar issues.
please review and merge if ok. thanks! 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,10 +89,10 @@ The list of ***benefits*** Phoenix has over
...
 [github.com/dwyl/learn-phoenix-framework#our-**top-10-reasons**-why-phoenix](https://github.com/dwyl/learn-phoenix-framework#our-top-10-reasons-why-phoenix) <br />
 
-+ **Elixir** is the _functional_ programming language used to
-build using the Phoenix framework. Elixir is a _beautiful_ language
++ **Elixir** is the _functional_ programming language used
+by the Phoenix framework. Elixir is a _beautiful_ language
 written _from scratch_ to be ***friendly, concise and efficient***.
-***Yes***, Elixir not a "_popular_" as JavaScript, Java, C# or PHP,
+***Yes***, Elixir not as "_popular_" as JavaScript, Java, C# or PHP,
 but its' adoption is growing rapidly and most importantly its'
```
Comment (jruts) [inline README.md]: the itst > it's

File: README.md [hunk 2]
```diff
@@ -42,7 +42,8 @@ we use for dwyl products and services (clients).
 Each element in our stack was _carefully_ selected based
 on it's individual merits. <br />
```
Comment (SimonLab) [inline README.md]: should it be "its" instead of "it's" ?

Comment (nelsonic) [inline README.md]: good question... 🤔

Comment (jruts) [inline README.md]: Correct @SimonLab . I was confusing it with _it is_.

Comment (jruts) [inline README.md]: Although the second one should be _it's_, no?

**Comment by nelsonic** [comment]
@jruts + @SimonLab I've "_fixed_" that section by _completely_ removing the `it's` or `its'` 😉

**Comment by jruts** [comment]
@nelsonic I reviewed, but I have no rights to merge! So someone else has to do it. Good Stuff.

**Comment by nelsonic** [comment]
@jruts dang! i've made you a contrib so you can `merge` future PRs!
@SimonLab or @iteles  feel free to merge this one! 👍

**Comment by jruts** [comment]
GH should not allow you to add someone as _assignee_ if that person has no rights to merge anyway :p

**Pull request merged**